src/player.cpp

```diff
@@ -11066,26 +11107,38 @@ void player::forced_dismount()
     } else {
         add_msg( m_debug, "Forced_dismount could not find a square to deposit player" );
     }
+    if( g->u.get_grab_type() != OBJECT_NONE ) {
```

Comment:
```suggestion
    if( get_grab_type() != OBJECT_NONE ) {
```
And below as well. All the other code in this functions refer to `this` implicitly, those lines here should not be an exception. Or is there a reason for it?

Reply:
Not sure of the reason, but it errors without. 

Which is weird, considering its in player namespace.

Reply:
Right, because it's an `avatar` member function, not a `player` member function. Keep it than.